src/formatDate.js

```diff
@@ -50,7 +50,60 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let result = [];
+
+  const partsDate = date.split(fromFormat[fromFormat.length - 1]);
+
+  const info = {};
+
+  for (let i = 0; i < fromFormat.length; i++) {
+    switch (fromFormat[i]) {
+      case 'YY':
+      case 'YYYY':
```

Comment:
Consider will be better if you create a variable for all magic words and use them everywhere